packages/ui/src/components/va-pagination/VaPagination.demo.vue

```diff
@@ -1,216 +1,376 @@
+                :pages="15"
+                :visible-pages="5"
+                border-color="primary"
+                gapped
+                rounded
+                preset="secondary"
+              />
+            </VbCard>
+          </td>
+        </tr>
+        <tr>
+          <td>Custom active page border</td>
+          <td>
+            <VbCard>
+              <va-pagination
+                v-model="activePage"
+                :pages="15"
+                :visible-pages="5"
+                preset="default"
+                active-page-color="warning"
+                gapped
+                rounded
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="15"
+                :visible-pages="5"
+                preset="primary"
+                active-page-color="warning"
+                gapped
+                rounded
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="15"
+                :visible-pages="5"
+                preset="secondary"
+                active-page-color="warning"
+                gapped
+                rounded
+              />
+            </VbCard>
+          </td>
+        </tr>
+        <tr>
+          <td>Sizes</td>
+          <td>
+            <VbCard>
+              <va-pagination
+                v-model="activePage"
                 :pages="10"
                 :visible-pages="3"
-                v-model="activePage"
+                size="small"
               />
               <va-pagination
-                size="large"
+                v-model="activePage"
                 :pages="10"
                 :visible-pages="3"
+                size="medium"
+              />
+              <va-pagination
                 v-model="activePage"
+                :pages="10"
+                :visible-pages="3"
+                size="large"
               />
             </VbCard>
           </td>
         </tr>
         <tr>
-          <td>Colored Pagination</td>
+          <td>Colored</td>
           <td>
             <VbCard>
               <va-pagination
                 v-model="activePage"
-                :visible-pages="4"
                 :pages="15"
+                :visible-pages="4"
                 color="info"
               />
               <va-pagination
                 v-model="activePage"
-                :visible-pages="4"
                 :pages="15"
+                :visible-pages="4"
                 color="danger"
               />
               <va-pagination
                 v-model="activePage"
-                :visible-pages="4"
                 :pages="15"
+                :visible-pages="4"
                 color="warning"
               />
             </VbCard>
           </td>
         </tr>
         <tr>
-          <td>Disabled Pagination</td>
+          <td>Disabled</td>
           <td>
             <VbCard>
               <va-pagination
```

Comment:
Except for cursor - disabled buttons have no distinction from active ones.

<img width="409" alt="image" src="https://user-images.githubusercontent.com/5394573/189233931-cd2b7369-f094-4c19-89ab-9b862c61e6ee.png">